Alignment/OfflineValidation/bin/DMRtrends.cc

```diff
@@ -219,10 +220,10 @@ int trends(int argc, char* argv[]) {
             gname.ReplaceAll(" ", "_");
             auto g = Get<TGraphErrors>(gname);
             assert(g != nullptr);
-            g->SetTitle(""); // for the legend
-            g->SetMarkerSize(0.6);
+            g->SetTitle(gtitle); // for the legend
+            g->SetMarkerSize(0.8);
             int color = alignment.second.get<int>("color");
-            int style = alignment.second.get<int>("style");
+	    int style = (int)(((double) alignment.second.get<int>("style"))/100.);
```

Comment:
Can you not just use `alignment.second.get<double>`? instead of all these conversions...